maintainers/maintainer-list.nix

```diff
@@ -10774,4 +10774,14 @@
     github = "benneti";
     githubId = 11725645;
   };
+  dschrempf = {
```

Comment:
Please sort your entry alphabetically.

Reply:
Done! I guess the confusion arose because the last 10 entries or so are not sorted, and I assumed I had to add myself at the end.